Upstream: yes
Reason: fix build

From 049113e719dd2fc4446d054fa1a3aada330094f0 Mon Sep 17 00:00:00 2001
From: Alex Merry <alex.merry@kde.org>
Date: Sat, 29 Mar 2014 12:10:23 +0000
Subject: [PATCH 2/2] Explicitly declare exported interfaces of libraries

This is (a) generally a good idea, and (b) necessary with CMake
2.8.12.0, which defaults to PRIVATE linkage (when CMP0022 is NEW) if no
keywords are given to target_link_libraries().
---
 plasma-desktop/applets/pager/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/plasma-desktop/applets/pager/CMakeLists.txt b/plasma-desktop/applets/pager/CMakeLists.txt
index ca2b88c..6e9ae36 100644
--- a/plasma-desktop/applets/pager/CMakeLists.txt
+++ b/plasma-desktop/applets/pager/CMakeLists.txt
@@ -19,5 +19,9 @@ target_link_libraries(pagerplugin
                       Qt5::DBus
                       taskmanager)
 
+if (X11_FOUND)
+    target_link_libraries(pagerplugin Qt5::X11Extras)
+endif()
+
 install(TARGETS pagerplugin DESTINATION ${QML_INSTALL_DIR}/org/kde/plasma/private/pager)
 install(FILES plugin/qmldir DESTINATION ${QML_INSTALL_DIR}/org/kde/plasma/private/pager)
-- 
1.9.1

